tests/unit/views/pacbio/PacbioRunIndex.spec.js

```diff
@@ -202,8 +202,10 @@ describe('Runs.vue', () => {
     })
 
     it('will filter the runs in the table', () => {
-      expect(wrapper.find('tbody').findAll('tr').length).toEqual(1)
-      expect(wrapper.find('tbody').findAll('tr').at(0).text()).toMatch(/Sequel I/)
+      wrapper.vm.$nextTick(() => {
+        expect(wrapper.find('tbody').findAll('tr').length).toEqual(1)
```

Comment:
Same comment about asserting inside the callback.  Failed assertions might not cause tests to fail.